RezaModule/view/frontend/templates/hello.phtml

```diff
@@ -1,6 +1,3 @@
-
-
-
 <h1>Signup page</h1>
 ///** @var \Signup\RezaModule\Hello  */
 <form action="<?php echo $this->getUrl("signup/index/save") ?>" method="post">
```

Comment:
- use short tag echo in php  <?= ?> 
- use escapeHtml function $block->escapeHtml 
or 
use /* @escapeNotVerified */ style before echo